misc-scripts/stable_id_lookup/populate_stable_id_lookup.pl

```diff
@@ -224,7 +224,7 @@ sub process_dbs {
 
   foreach my $db (@dbs) {
 
-    if ( $db =~ /([\w\_]+)_(core|otherfeatures)_([\d\_\w]+)/ )
+    if ( $db =~ /([a-z]+_[a-z0-9]+(?:_[a-z0-9]+)?_([a-z]+)(?:_\d+)?)_(\d+)_\w+/ )
```

Comment:
As far as I can tell, this depends on the `like "%$version%"` clause in the show query to work correctly. That said, it parses out the correct things on the database names in e! and e!g databases.